docsrc/imap/developer/jmap.rst

```diff
@@ -37,50 +37,83 @@ Test JMAP support
 
 Once Cyrus is running, you can test JMAP on the command line for any existing Cyrus user. The user must at least have an INBOX provisioned but is not required to have any calendars, contacts or messages.
 
-To obtain the JMAP calendars for user ``test``, issue the following request:
+To obtain the JMAP mailbox folders for user ``test``, issue the following request:
 
 .. code-block:: bash
 
     curl -X POST \
          -H "Content-Type: application/json" \
          -H "Accept: application/json" \
          --user test:test \
-         -d '[["getCalendars", {}, "#1"]]' \
+         -d '{
+           "using": [ "urn:ietf:params:jmap:core", "urn:ietf:params:jmap:mail" ],
+           "methodCalls": [[ "Mailbox/get", {}, "c1" ]]
+         }' \
          http://localhost/jmap
```

Comment:
Could you also update the endpoint from `http://localhost/jmap` to `http://localhost/jmap/` please? (i.e. add a trailing '/')

Without it, you'll get a "301 Moved Permanently" response, which curl will ignore (unless invoked with `--location`, but the example doesn't set this flag, and it seems better to accustom people to the real path and not rely on the redirect)

Reply:
Oopsie. Nice catch :+1:
I updated the pull request.